sample_workloads/nccltest/docker/scripts/container_entry.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+# Container entry script. Runs NCCL-level benchmarks and report results.
+
+set -u
+
+# Job parameters.
+: "${JOB_TIMESTAMP:?Must set JOB_TIMESTAMP}"
+: "${JOB_NAME:?Must set JOB_NAME}"
+: "${MASTER_ADDR:?Must set MASTER_ADDR}"
+: "${NNODES:?Must set NNODES}"
+: "${NODE_RANK:?Must set NODE_RANK}"
+
+# GCS bucket to be used.
+: "${GCS_BUCKET:?Must set GCS_BUCKET}"
```

Comment:
Do we want this to be GCS dependent? Do we expect customers to create a GCS bucket just to run this? A mount failure will not be fatal, but just checking.